### ISSUE TITLE
Feature: Support disabling delation of RDS automated backups

### ISSUE THREAD

**Issue by nathanielks** [terraform-providers/terraform-provider-aws#6507]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

In [a recent announcement](https://aws.amazon.com/about-aws/whats-new/2018/11/amazon-rds-automated-backups-can-now-be-retained/), AWS said you can now save RDS backups after the instance itself has been deleted. This behavior is opt-in and needs to have support for it in Terraform.

### New or Affected Resource(s)

* aws_db_instance

### Potential Terraform Configuration

```hcl
resource "aws_db_instance" "main" {

  delete_automated_backups = false
  // Additional config...
}
```

### References

N/A

**Comment by sunilkumarmohanty** [comment]
I think aws sdk for go  does not support this feature yet.

**Comment by bflad** [comment]
Support for this functionality has been merged and will release with version 2.49.0 of the Terraform AWS Provider, Thursday this week. Thanks to @MrLunar for the implementation. 👍

**Comment by alex** [comment]
Huge thanks to both you and @MrLunar for bringing this to completion!
